### ISSUE TITLE
VariableGroup doesn't work with key_vault parameter (2)

### ISSUE THREAD

**Issue by ringods** [pulumi/pulumi-azuredevops#156]
### What happened?

Same as in #52, which was fixed in v2.5.0 but reappeared again since version 2.6.0 of the provider.

### Expected Behavior

Same as in #52

### Steps to reproduce

Same as in #52

### Output of `pulumi about`

Same as in #52, but with v2.6.0 or v2.8.0 of the provider.

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by t0yv0** [comment]
Thank you for reporting this! Recent versions of the bridge moved validation to the CheckConfig method and I suspect validations were disabled for a while because Configure method does not report errors; hence this issue resurfaced.

I was having a quick look @ringods but hitting a few snags. The creds in `Pulumi Bot - Azure DevOps`  1Password seem to complain

```
       * According to Azure Active Directory, your Identity 27d5a3ac-ce94-4bbb-a257-f1a28704a292 is currently Deleted within the following Azure Active Directory: pulumi.com. Please contact your AAD administrator to resolve this.  For more information please read https://go.microsoft.com/fwlink/?linkid=2007012
```

I'm asking around. Do you have a working set of test credentials to repro?

The second issue is that the program is very fragmentary. Do you have a bit of a more fleshed out example? If not I can try to manufacture something but it is always a time-saver to have a complete repro.

**Comment by VenelinMartinov** [comment]
This was fixed with https://github.com/pulumi/pulumi-terraform-bridge/pull/1583 and released in 2.14.2
